container/root/etc/php/7.0/mods-available/xdebug.ini

```diff
@@ -0,0 +1,23 @@
+; Enable The Extension
+;zend_extension=xdebug.so
+
+; Remote Debugging
+xdebug.remote_enable=on
+xdebug.remote_handler=dbgp
+xdebug.remote_mode=req
+; xdebug.remote_host=192.168.59.3
+xdebug.remote_connect_back=on
+; This is for EVERY request (if set to 1)
+xdebug.remote_autostart=0
+
+; Nesting level (updated for Behat)
+xdebug.max_nesting_level=256
+
+; Profiling
+xdebug.profiler_output_dir=/tmp/cachegrind
```

Comment:
@jimdelois what do you think of making this `/app/cachegrind` (or some other place that is gitignored but mounted)? Since it will be mounted locally, it will have enough space to contain the logs (xdebug does nothing if there's not enough; no warning or anything [and this takes hours to debug; which I can say from experience]) and it will be easier to extract from the container.